src/internal/metrics.ts

```diff
@@ -98,22 +107,21 @@ const findAWSC = (currentWindow?: MetricsWindow): AWSC | undefined => {
   }
 };
 
-// react is the default framework we're logging, for angular we need to set it explicitly
-let framework = 'react';
-function setFramework(fwk: string) {
-  framework = fwk;
-}
-
 export const Metrics = {
-  initMetrics(fwk: string) {
-    setFramework(fwk);
+  initMetrics(theme: string) {
+    setTheme(theme);
   },
 
   /**
    * Calls Console Platform's client logging JS API with provided metric name, value, and detail.
    * Does nothing if Console Platform client logging JS is not present in page.
    */
   sendMetric(metricName: string, value: number, detail?: string): void {
+    if (!theme) {
+      // Metrics need to be initialized first (initMetrics)
+      return;
```

Comment:
We could also `console.error` this.

Reply:
If our code works correctly, this condition should never be visited. If any reporting, it should be throwing an error (fine as a future PR)

Reply:
All the other error cases in this function (and file, actually) use `console.error` and don't throw. So I would be in favor of another `console.error`.

Reply:
ok